lib/react_on_rails/server_rendering_pool.rb

```diff
@@ -48,6 +48,9 @@ def self.server_render_js_with_console_logging(js_code)
       result
     end
 
+    def self.before_render(component_name, props, prerender_options, request); ""; end
```

Comment:
Yeah, they idea would be to Monkey patch it to override these methods.

Another option would be to have a default class which had these methods and then if you wanted to implement them then extend that class and tell react_on-rails to use it through configuration.